packages/odyssey-storybook/src/components/odyssey-mui/DataTable/DataTable.stories.tsx

```diff
@@ -868,3 +868,37 @@ export const CustomFilterWithDefaultVariant: StoryObj<DataTableProps> = {
     );
   },
 };
+
+export const Truncation: StoryObj<DataTableProps> = {
+  render: function C() {
+    const columns = useMemo(
+      () => [
+        {
+          accessorKey: "truncated",
+          header: "Truncated",
+        },
+        {
+          accessorKey: "wrapped",
+          header: "Wrapped",
+          enableWrapping: true,
+        },
+      ],
+      [],
+    );
+
+    const getData = useCallback(() => {
+      const data = [];
+      for (let i = 0; i < 10; i++) {
```

Comment:
Little bit of a nit/subjective opinion. I prefer something like... 
```suggestion
      [...Array(10)].forEach(() => 
```